### ISSUE TITLE
Apply CSS styles for new component workflow - User can now add new fields automatically to allow for multiple entries of data.

### ISSUE THREAD

**Issue by MissionT** [bcgov/digital-journeys#765]
The form designer can now add new functionality to a form to allow users to add additional sets of data such as multiple entries of a person (name, email,  etc)for example.  

New process- Form's designer: Add a data grid to the form, add a field set within the data grid and place fields within the component.
The result is a column or vertical row of fields that will repeat after the selection has been made by the user to add more fields.

Can the devs please add CSS/Styles to these components for the form to blend into the current form's style, look or feel? 
Below is an example of the 1.10 Complaint form. The product owner would like the user to be able to add up to 50 groups of fields.  This is related to ticket: 759 

![image.png](https://images.zenhubusercontent.com/62eafeafcc2f007b1446cbd1/b9959136-a4b7-4718-8cde-ab79ab6c901c)

**Comment by bhumin-fw** [comment]
Hey Team,
I have overridden some CSS for this Datagrid and fieldset component. Please check below screenshots.
**"Add new" as button**
![Screenshot 2022-12-01 at 9.42.23 PM.png](https://images.zenhubusercontent.com/62e98579ff9c075848ccdb67/14138d15-21e8-48fc-810b-6656a57c14e0)
![Screenshot 2022-12-01 at 9.44.20 PM.png](https://images.zenhubusercontent.com/62e98579ff9c075848ccdb67/d1d35b64-fa54-4911-b906-0a83a901f025)

**"Add new" as a link**
![Screenshot 2022-12-01 at 9.47.43 PM.png](https://images.zenhubusercontent.com/62e98579ff9c075848ccdb67/abf76edc-8454-49e0-9abb-5e17f00ca856)

**Comment by MeghanStothers** [comment]
We looked as a group yesterday - will report today

**Comment by MissionT** [comment]
We discussed this as group and first need to know if the behavior of any changes in the code will affect all forms globally, or if the code is local to each form. 

Can we get feedback on the following:

 The delete button: its located at the far right as very small circle containing an  x, which is not very intuitive. Can we have a large button labelled 'Remove'.   For flexibility it would be nice if the forms designer can edit this label if need be to accomodate different forms. 
Note: The button button  '+Add another respondent' label can be manually changed by the forms designer so this does not need to be looked at.  

Overall,  we like the new format.  The new job aid should be reviewed by the dev to make sure the instructions are correct since an update with styling as been added.
https://bcgov.sharepoint.com/:b:/t/03991/EUauCqwlkZFLkiCown4ZvgkBu0NDRC_-UgjM5KZ4HhdYtg?e=UImmCC
Thanks, Chris

**Comment by bhumin-fw** [comment]
@MissionT I'm not able to open or access this document. Could you please share access to it with me or attach it here?

**Comment by MissionT** [comment]
I sent an email to Bhumin re the small change we would like. Thanks Chris

**Comment by bhumin-fw** [comment]
@MissionT @MeghanStothers @fazil-ey 

I have added a text button in place of the "x" icon. (Text can be changed by the developer only and applied to all forms with the same data grid and fieldset component combination only).

Added the same button in 2 different places. One with top-right and one with bottom-right in the section. checkout the below images.
![Screenshot 2022-12-06 at 6.30.36 PM.png](https://images.zenhubusercontent.com/62e98579ff9c075848ccdb67/7b9b34a9-fc00-495c-9bb1-d3b81b519eaa)
![Screenshot 2022-12-06 at 6.29.04 PM.png](https://images.zenhubusercontent.com/62e98579ff9c075848ccdb67/0ce4f894-fa49-4a42-83ba-b3b1305426b4)
